docs/usage.rst

```diff
@@ -4,9 +4,135 @@ Usage
 Creating Virtual Machines
 -------------------------
 
-Create vms
+Follow the steps carefully so that you dont get any error while creating virtual machine. The steps are very easy only need is to follow them in order.
+
+++++++
+Step 1
+++++++
+
+First you need to create compute resource.
+
+To do so click on Infrastructure -> Compute Resource
+
+* Fill all the details
+* Make sure there is no repeatation of name, remote system is reachable from your system and root password is correct.
+* All the fields have validation.
+
+++++++
+Step 2
+++++++
+
+Once done with compute resource move to profiles.
+
+Click on Infrastructure -> Profile
+
+* Name your Profile, fill the fields of RAM, VCPUS and Disk Space
+* Name should not get repeated.
+
+Once you click *Submit* Profile gets added to database and is visible in **View Existing** Section.
+
+++++++
+Step 3
+++++++
+
+Now, you have to add Packages for you Virtual machine.
+
+* Click on Content -> Product
+
+	Enter the name of package and give URL from where it should fetch the package repository.
+
+* Click on Content -> View
+
+	Enter the name for View and select the product from the list avialable. You can select one or more products to encapsulate into single View.
+
+* Click on Content -> Activation key
+
+	Here enter the name for your activation key and select one or more Views of your desire.
+
+.. note::
+
+	Activation is very essential at time of creating host.
```

Comment:
Activation is **not** essential, without activation virtual machines can be created
It is optional